frontend_tests/zjsunit/index.js

```diff
@@ -16,6 +16,7 @@ global._ = require('underscore/underscore.js');
 const _ = global._;
 
 global.Dict = require('../../static/js/dict').Dict;
+global.FoldDict = require('../../static/js/fold_dict').FoldDict;
```

Comment:
I’d prefer to keep `FoldDict` out of `global` and import it just in the files where it’s used. (`Dict` is special because it’s used in `zjsquery.js`—although really I’d like to get it out of `global` too along with everything else, it’s just a bit more work.)